src/libs/show-names.js

```diff
@@ -16,6 +16,10 @@ const addUsersName = (user, name) => {
 			$userLink.after(`<span class="comment-full-name">(${name})</span>`);
 		}
 		$userLink.addClass('has-full-name');
+		const $header = $userLink.parent().parent();
+		if ($header.hasClass('timeline-comment-header-text')) {
+			$header.html($header.html().replace('commented', ''));
+		}
```

Comment:
`.html(.html())` regenerates all the elements in it. It's best to only remove the text node:

```js
userLink.parentNode.nextSibling.remove();
```